react-ui/package.json

```diff
@@ -1,6 +1,24 @@
 {
-    "name": "connect-me-website",
```

Comment:
Is there a specific reason why the name and description was changed?

Reply:
changed it a little to make it more appropriate